devicetypes/smartthings/smartweather-station-tile.src/smartweather-station-tile.groovy

```diff
@@ -22,9 +22,21 @@ metadata {
         capability "Temperature Measurement"
         capability "Relative Humidity Measurement"
         capability "Ultraviolet Index"
+        capability "Wind Speed"
+        capability "stsmartweather.Wind Speed" // "Wind Speed" only supports m/s unit, however we want to create both events
+        capability "stsmartweather.Wind Direction"
+        capability "stsmartweather.Apparent Temperature"
+        capability "stsmartweather.Astronomical Data"
+        capability "stsmartweather.Percipitation"
```

Comment:
sp: Precipitation here and other places

Reply:
Yup, thanks. That was causing an exception that I was facing. A conversion handler deep in the bowels of one of our services was trying to look up a misspelled version of the attribute from a Map using the attr name as the key and value was returned as `null`. Without checking it tries to execute commands on `null` and throw an NPE instead of either gracefully exiting or throwing a more meaningful exception (`CapabilityAttributeNotValid` maybe? heh). That was painful to track down...